buildlogger/logger_producer.go

```diff
@@ -10,6 +10,7 @@ import (
 // options.LoggerProducer for BuildloggerV3.
 type JasperLoggerOptions struct {
```

Comment:
Just trying to understand: how does this get used? I don't see cedar vendored in jasper

Reply:
Nope, it is an injectable logger and vendored in curator and added to the jasper logger producer registry there: https://github.com/mongodb/curator/blob/master/cmd/curator/curator.go#L75-L97.

Reply:
neat! resmoke uses jasper through curator?

Reply:
Yep!